easybuild/easyconfigs/h/Hadoop/Hadoop-2.4.0-seagate-722af1-native.eb

```diff
@@ -21,6 +21,8 @@ builddependencies = [
 ]
 dependencies = [(java, javaver)]
 
+patches = [('Hadoop-TeraSort-on-local-filesystem.patch', 0)]
```

Comment:
no need to remove ambiguity, kick it out since `eb` usually guesses the patch level correctly